src/main/java/com/aws/greengrass/deployment/ShadowDeploymentListener.java

```diff
@@ -135,6 +136,14 @@ private void subscribeToShadowTopics() {
                         (e) -> logger.atError().log("Error processing updateShadowResponse", e))
                         .get(TIMEOUT_FOR_SUBSCRIBING_TO_TOPICS_SECONDS, TimeUnit.SECONDS);
                 logger.info("Subscribed to update named shadow accepted topic");
+
+                iotShadowClient.SubscribeToUpdateNamedShadowRejected(updateNamedShadowSubscriptionRequest,
+                        QualityOfService.AT_LEAST_ONCE,
+                        updateShadowRejected -> handleNamedShadowRejectedEvent(),
+                        (e) -> logger.atError().log("Error processing named shadow update rejected response", e))
+                        .get(TIMEOUT_FOR_SUBSCRIBING_TO_TOPICS_SECONDS, TimeUnit.SECONDS);
+                logger.info("Subscribed to update named shadow rejected topic");
+
```

Comment:
line 152: why is it `QualityOfService.AT_MOST_ONCE`?

Reply:
mybad, updated to  QualityOfService.AT_LEAST_ONCE,